src/seq/Seq.jl

```diff
@@ -138,8 +138,7 @@ export
     AA_Gap,
     FASTA,
     FASTASeqRecord,
-    FASTQReader,
-    FASTQWriter,
+    FASTQ,
     FASTQSeqRecord,
```

Comment:
FASTQSeqRecord and FASTASeqRecord to be removed? Depends on whether the old IO system should be included with the final monolithic 0.5 or not.